perf/collector_libpfm_test.go

```diff
@@ -20,10 +20,12 @@ package perf
 import (
 	"bytes"
 	"encoding/binary"
-	"github.com/stretchr/testify/assert"
 	"testing"
 
+	"github.com/stretchr/testify/assert"
```

Comment:
Why is this import in separated group? It seems that it can be in group with `info "github.com/google/cadvisor/info/v1"` and `"github.com/google/cadvisor/stats"`.

Reply:
It's from a different vendor?